src/main/scala/chisel/lib/dclib/DCOperator.scala

```diff
@@ -0,0 +1,42 @@
+package chisel.lib.dclib
+
+import chisel3._
+import chisel3.util._
+
+/**
+ * This module declares a multi-input decoupled operator.  This is an example using showing how to use
+ * DCInput and DCOutput modules to create a module with registered-output timing.
+ *
+ * @param n      The number of inputs for the operator
+ * @param width  The bit-width of inputs and outputs
+ * @param op     Function with the required operator
+ */
```

Comment:
Other modules use Javadoc style with gutter alignment on column 2. For consistency, this PR could use this here, too. Also, consider adding cross-links: 
```suggestion
/**
  * This module declares a multi-input decoupled operator.  This is an example using showing how to use
  * [[DCInput]] and [[DCOutput]] modules to create a module with registered-output timing.
  *
  * @param n      The number of inputs for the operator
  * @param width  The bit-width of inputs and outputs
  * @param op     Function with the required operator
  */
```